README.md

```diff
@@ -3,8 +3,37 @@
 
 # dwave-scikit-learn-plugin
```

Comment:
```suggestion
# D-Wave `scikit-learn` Plugin
```

We do mostly use the package name as the title here but where the package name is less intuitive we make exceptions, for example [D-Wave NetworkX](url). This package name does not quite roll off one's tongue.